composer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "automattic/vip-go-skeleton",
+  "description": "Used for setting up development dependencies across the whole repository.",
+  "homepage": "https://github.com/automattic/vip-go-skeleton",
+  "license": "GPL-2.0-or-later",
+  "require": {
+    "php": ">= 7.4"
+  },
+  "require-dev": {
+    "automattic/vipwpcs": "^2.3",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
+    "phpcompatibility/phpcompatibility-wp": "^2"
+  },
+  "scripts": {
+    "install-codestandards": "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run"
```

Comment:
I can't remember the last time I ran this manually, so we can probably keep things simple and remove this scripts section.